salt/api-gateway/config/lib-systemd-system-kong.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description="kong, api-gateway"
+After=network.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+StandardError=syslog
```

Comment:
this is actually a pretty good introduction to systemd in general, but there is a section called 'Systemd Journal Basics' not too far down that explains how the output from services are handled these days: https://www.loggly.com/ultimate-guide/linux-logging-with-systemd/